tests/performance.php

```diff
@@ -1,44 +1,50 @@
 <?php
-require '../lib/SqlFormatter.php';
+
+declare(strict_types=1);
+
+require '../vendor/autoload.php';
+
+use Doctrine\SqlFormatter\SqlFormatter;
 
 //this is the default value
 //set to '0' to disable caching
 //a value between 10 and 20 seems to give the best result
-SqlFormatter::$max_cachekey_size = 15;
+SqlFormatter::$maxCachekeySize = 15;
```

Comment:
This will be a difference in the "API" compared to the original SQLFormatter.

Reply:
Indeed, and you are correct with your approval message, there are going to be other BC-breaks, now is the time to make them.